OutOfSchool/OutOfSchool.BusinessLogic/Models/WorkshopDraft/WorkshopDraftFilterAdministration.cs

```diff
@@ -1,12 +1,25 @@
 ﻿using OutOfSchool.BusinessLogic.Models.Workshops;
 using OutOfSchool.BusinessLogic.Util.CustomValidation;
 using OutOfSchool.Services.Enums.WorkshopStatus;
-using System.ComponentModel.DataAnnotations;
 
 namespace OutOfSchool.BusinessLogic.Models.WorkshopDraft;
 public class WorkshopDraftFilterAdministration : WorkshopFilterAdministration
 {
-    [EnumDataType(typeof(WorkshopDraftStatus), ErrorMessage = Constants.EnumErrorMessage)]
```

Comment:
need to think about how to do this check for enum datatype, because it is possible in some cases to pass an array of random integers and this will deserialize into broken enums.